Content/1.1.3 - PreAuthorization Review/preAuthorizationReview.json

```diff
@@ -0,0 +1,20 @@
+{
+	"preAuthorizationReview_title": "Review your Pre-Authorization",
+	"preAuthorizationReview_subtitle": "Proposed by %@",
+	"preAuthorizationReview_loading": "Incoming Pre-Authorization",
+	"preAuthorizationReview_fees_title": "Pre-authorization will be returned to %@ for processing.",
+	"preAuthorizationReview_fees_subtitle": "Network fees will be paid by the dApp",
+	"preAuthorizationReview_expiration_atTime": "Valid for the next **%@**",
+	"preAuthorizationReview_expiration_afterDelay": "Valid for **%@ after approval**",
+	"preAuthorizationReview_expiration_expired": "Pre-authorization is no longer valid",
+	"preAuthorizationReview_timeFormat_day": "1 day",
+	"preAuthorizationReview_timeFormat_days": "%d days",
+	"preAuthorizationReview_timeFormat_hour": "%@ hour",
+	"preAuthorizationReview_timeFormat_hours": "%@ hours",
+	"preAuthorizationReview_timeFormat_minute": "%@ minute",
+	"preAuthorizationReview_timeFormat_minutes": "%@ minutes",
+	"preAuthorizationReview_timeFormat_second": "1 second",
+	"preAuthorizationReview_timeFormat_seconds": "%d seconds",
+	"preAuthorizationReview_slideToSign": "Slide to sign and return",
```

Comment:
in zeplin the words are capitalized: "Slide to Sign and Return"

Reply:
good catch! updating